src/test/tgen/fixed_duration/verify.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+printf "TGen configuration: network=$1 client=$2\n"
+network=$1
+client=$2
+
+# read speed of server is the effective write speed of client
+# read speed of client is the effective write speed of server
+# tail -1 gets the fastest speed reached
+server_bps_read="$(grep -r --include="server.tgen.*.stdout" "driver-heartbeat" \
+    | cut -d' ' -f8 \
+    | cut -d'[' -f2 \
+    | cut -d',' -f1 \
+    | cut -d'=' -f2 \
+    | sort -n \
+    | tail -1)"
+client_bps_read="$(grep -r --include="client.tgen.*.stdout" "driver-heartbeat" \
```

Comment:
Maybe rename to `Bps`? We use `bps` elsewhere in shadow to refer to "bits-per-second".